src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -314,13 +342,44 @@ function hideResponseCommentAddForm(recipientIndex, giverIndex, qnIndx) {
     removeFormErrorMessage($('#button_save_comment_for_add' + id));
 }
 
-function showResponseCommentEditForm(recipientIndex, giverIndex, qnIndex, commentIndex) {
+function composeResponseCommendEditFormId(indexes, isIncludeSections) {
+    var sectionIndex;
+    var recipientIndex;
+    var giverIndex;
+    var qnIndex;
+    var commentIndex;
     var id;
+
+    if (isIncludeSections) {
+        sectionIndex = indexes[0];
+        recipientIndex = indexes[1];
+        giverIndex = indexes[2];
+        qnIndex = indexes[3];
+        commentIndex = indexes[4];
+    } else {
+        recipientIndex = indexes[0];
+        giverIndex = indexes[1];
+        qnIndex = indexes[2];
+        commentIndex = indexes[3];
+    }
+
     if (giverIndex || qnIndex || commentIndex) {
-        id = '-' + recipientIndex + '-' + giverIndex + '-' + qnIndex + '-' + commentIndex;
+        if (isIncludeSections) {
+            id = '-' + sectionIndex + '-' + recipientIndex + '-' + giverIndex + '-' + qnIndex + '-' + commentIndex;
+        } else {
+            id = '-' + recipientIndex + '-' + giverIndex + '-' + qnIndex + '-' + commentIndex;
+        }
     } else {
         id = '-' + recipientIndex;
     }
+
+    return id;
+}
+
+function showResponseCommentEditForm(/* firstIndex, secondIndex, thirdIndex, fourthIndex, fifthIndex */) {
+    var isIncludeSections = arguments.length === 5;
```

Comment:
Same for this. I would prefer if the arguments are named. This way of accessing function parameters will make it very difficult for any future maintainer. For the sectionIndex, pass an object opt containing sectionIndex as a field. We should do it in this PR because doing this requires context of what the old arguments were, and a future developer taking this as a new issue would not have knowledge of this.